### ISSUE TITLE
not actuale reposipory

### ISSUE THREAD

**Issue by romanown** [TheComputerM/svelte-materialify#167]
i forked this repository but it have not last changes. example `on:change` in Select component  svelte-materialify/packages/svelte-materialify/src/components/Select/Select.svelte . i have to try changing it to properly work with value and name.

**Comment by Florian-Schoenherr** [comment]
I don't know what/when you forked, but there it is:
https://github.com/TheComputerM/svelte-materialify/blob/628ef6a6b6056b6b81775da265708dcd9d962488/packages/svelte-materialify/src/components/Select/Select.svelte#L30
😄

**Comment by Florian-Schoenherr** [comment]
yes, as you can see that was closed and reimplemented like I have linked. This was never on the master branch.
